Makefile

```diff
@@ -4,34 +4,47 @@ build-prod:
 build-dev:
 	docker build -t openslides-auth-dev -f Dockerfile.dev .
 
-run-bash: | build-dev
-	docker-compose -f docker-compose.dev.yml up -d
-	docker-compose -f docker-compose.dev.yml exec auth wait-for-it auth:9004
-	docker-compose -f docker-compose.dev.yml exec auth bash
-	docker-compose -f docker-compose.dev.yml down
+build-test:
+	docker build -t openslides-auth-dev -f Dockerfile.test .
 
-run-check-cleanup: | build-dev
-	docker-compose -f docker-compose.dev.yml up -d
-	docker-compose -f docker-compose.dev.yml exec auth wait-for-it auth:9004
-	docker-compose -f docker-compose.dev.yml exec auth npm run lint-check
-	docker-compose -f docker-compose.dev.yml exec auth npm run prettify-check
-	docker-compose -f docker-compose.dev.yml down
+run-dev: | build-dev
+	docker-compose -f docker-compose.dev.yml up
+	stop-dev
 
-run-cleanup: | build-dev
+run-pre-test: | build-test
 	docker-compose -f docker-compose.dev.yml up -d
-	docker-compose -f docker-compose.dev.yml exec auth wait-for-it auth:9004
-	docker-compose -f docker-compose.dev.yml exec auth npm run cleanup
+	docker-compose -f docker-compose.dev.yml exec -T auth ./wait-for.sh auth:9004
+
+run-bash: | run-pre-test
+	docker-compose -f docker-compose.dev.yml exec auth sh
 	docker-compose -f docker-compose.dev.yml down
 
-run-tests: | build-dev
+run-check-lint: | run-pre-test
```

Comment:
all these dependencies will duplicate all dependent commands in the CI, so when running `run-check-lint`, `run-pre-test`, and `build-test` are also invoked again.

Reply:
What exactly do you mean? What should be changed in your opinion?

Reply:
Either do not use these dependencies (just remove them), or duplicate them and prepend "ci-" or so.

If you only use them in the CI and not your dev workflow the first option is better. If you use them manually often the second one is preferable.